pkg/dryrun/cmd.go

```diff
@@ -77,6 +78,13 @@ func (d *DryRunner) GetCmd() *cobra.Command {
 		"Disables colored output. By default, output is printed with colors.",
 	)
 
+	cmd.Flags().BoolVar(
+		&d.fullDiffs,
+		"full-diffs",
+		false,
+		"Enable this to display the full diff in the output. By default, the maximum number of diff lines shown is 53.",
```

Comment:
It's 50 lines of diff, plus a 3 line header... So I'd rather we say 50 lines, or just something like "By default, the number of lines are limited for readability" (and not put a specific number)